docs/ruleset_functions/normalize_interior_lighting_schedules.md

```diff
@@ -30,7 +31,7 @@ Logic:
 
     - Add lighting power per area to space total: `space_total_power_per_area += power_per_area`
 
-  - Check if lighting uses schedule to model occupancy control, get occupancy control credit: `if interior_lighting.are_schedules_used_for_modeling_occupancy_control: control_credit = data_lookup(table_G3_7, space.lighting_space_type, interior_lighting.occupancy_control_type)`
+  - Check if adjust_for_credit is true and lighting uses schedule to model occupancy control, get occupancy control credit: `if adjust_for_credit and interior_lighting.are_schedules_used_for_modeling_occupancy_control: control_credit = data_lookup(table_G3_7, space.lighting_space_type, interior_lighting.occupancy_control_type)`
 
   - Else, set occupancy control credit to 0: `else: control_credit = 0`
 
```

Comment:
I think we also need to define bonus_adjustment variable =1.25 if InteriorLighting.occupancy_control_type=PARTIAL_AUTO_ON OR MANUAL ON, and =1 otherwise, and then in rows 40 and 44 update the forumal to hourly_use_per_area_array.append(hourly_value / (1 - control_credit***bonus_adjustment**) *....

I see now that your note 1 suggests to add column to Table 3.7 to account for bonus adjustment, but I am not sure how this will work because we need to know for each interior lighting object whether it has OS controls that qualify it for this additional credit.

Reply:
The note might be confusing but the function [table_G3_7_fns](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/data_fns/table_G3_7_fns.py#L167-L175) takes care of the control credit bonus logic and the [database ](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/data/ashrae_90_1_prm_2019.prm_interior_lighting.json#L1246-L1248)we are using have the bonus credit.

Note 1 was updated by Xing 11 months ago. I can remove it since we do not have this issue anymore.?